src/steps/owner.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  createIntegrationEntity,
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+} from '@jupiterone/integration-sdk-core';
+import { createAPIClient } from '../client';
+import { IntegrationConfig } from '../config';
+
+export async function fetchOwners({
+  jobState,
+  instance,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  const apiClient = createAPIClient(instance.config);
+  await apiClient.iterateOwners(async (owner) => {
+    await jobState.addEntity(
+      createIntegrationEntity({
+        entityData: {
+          source: {
+            id: owner.id,
+          },
+          assign: {
+            _key: `hubspot-owner-${owner.id}`,
```

Comment:
Can you make the key look like this: `hubspot_owner:${owner.id}`? I understand that inside different repositories I've shared with you, the format slightly changes, but we were using underscores so far to separate resource type from the platform name and `:` to separate between the "name" part and the unique identifier so it'd be great if we can stick with that.